thorclient/httpclient/client_test.go

```diff
@@ -300,6 +301,25 @@ func TestClient_GetTransaction(t *testing.T) {
 	assert.Equal(t, expectedTx, tx)
 }
 
+func TestClient_CallTransaction(t *testing.T) {
```

Comment:
can we also add negative tests ?

Reply:
I guess we can but those are mock tests afaik. So not sure what the benefit of returning a hardcoded "error" response will be and checking if its what we expect.